LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 KA Extension Developers
+Copyright (c) 2023 KA Extension Developers
```

Comment:
:thinking:

Reply:
I figure its better to be ahead of the game given I don't want to go back and update it later.

Reply:
But it's wrong...